File: AkitaModel/BgNhis/NhifSpeciality.cs
namespace Skyware.Lis.AkitaModel.BgNhis;

/// <summary>
/// NHIF/NHIS (НЗОК/НЗИС) speciality.
/// </summary>
public class NhifSpeciality
{

    /// <summary>
    /// Code, according no NHIF (НЗОК).
    /// </summary>
    public string NhifCode { get; set; }

    /// <summary>
    /// Code, according to NHIS (НЗИС) CL006.
    /// </summary>
    public string NhisCode { get; set; }

    /// <summary>
    /// Name of the speciality, according to NHIF (НЗОК).
    /// </summary>
    public string Name { get; set; }

}

File: AkitaMotelTS/src/BgNhis/NhifSpeciality.ts
/**
 * Skyware.Lis.AkitaModel.BgNhis
 */

/**
 * NHIF/NHIS (НЗОК/НЗИС) speciality.
 */
export class NhifSpeciality {
  /**
   * Code, according to NHIF (НЗОК).
   */
  nhifCode?: string;

  /**
   * Code, according to NHIS (НЗИС), set CL006.
   */
  nhisCode?: string;

  /**
   * Name of the speciality, according to NHIF (НЗОК).
   */
  name?: string;
}
